Tasks_03:

DmytriiTsiupko opened the pull request:
1. Create a program that will expect the user to enter text and output information for each typed character:

- this "number" + what it is (even, odd),
- this is a "letter" + what is it (uppercase or lowercase),
- is a "symbol"

2. Create a program that will print “I love Python” to the console every 4.2 seconds until its execution is manually interrupted.

DmytriiTsiupko committed:
Tasks:
1. Create a program that will expect the user to enter text and output information for each typed character:

- this "number" + what it is (even, odd),
- this is a "letter" + what is it (uppercase or lowercase),
- is a "symbol"

2. Create a program that will print “I love Python” to the console every 4.2 seconds until its execution is manually
interrupted.

klymenok commented on Lesson_03/solution_task_n1.py:
```diff
@@ -0,0 +1,33 @@
+            else:
+                print(f"and it's an odd number")\
+
+        elif char.isalpha() == True:
```
You can skip `== True` since if will make this check in the end. So it can be simplified to `elif char.isalpha():`

klymenok commented on Lesson_03/solution_task_n1.py:
```diff
@@ -0,0 +1,33 @@
+                print("and it's a small letter")
+
+        else:
+            if char.isdigit() != True and char.isalpha() != True:
```
This `if` statement is not needed since we can assume that everything that is not a digit and is not a letter is a symbol.

klymenok commented on Lesson_03/solution_task_n2.py:
```diff
@@ -0,0 +1,10 @@
+
+while True:
+
+    time_out = time.sleep(4.2)
```
You can use it simply `time.sleep(4.2)`. There is no need to save the result in `time_out` variable

klymenok reviewed (changes requested):
Solution is good, but I would like to ask you make some changes to make it looks better

DmytriiTsiupko committed:
Pull Request changes

In solution_task_n1:
- Skipped '== True'
- Deleted 'if' statement

In solution_task_n2.py:
-Deleted variable for

DmytriiTsiupko commented:
In solution_task_n2.py:
-Deleted variable for time.sleep(4.2)

klymenok reviewed (approved):
Looks good!
